flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": [
+    "x86_64"
```

Comment:
why?

Reply:
At the moment we will only support the x86_64 architecture.

Reply:
why? it's just an electron app. I thought the goal of using electron was to be cross-platform.

Reply:
Although it is just an Electron application, we here at Wiselabs do not have the machinery and available time to commit to a quality version for 3 operating systems across 2 different architectures. Currently, we are able to test and ensure the quality of this application on Linux x86_64, macOS x86_64 and ARM, and Windows x86_64. As simple as it may seem, it is a matter of accountability.